src/main/java/com/microsoft/azure/springcloudplayground/service/ServiceNames.java

```diff
@@ -67,4 +67,17 @@ public static Service toService(@NonNull MicroService microService) {
                         .build();
         }
     }
+
+    public static boolean isAzureService(@NonNull String serviceName) {
+        switch (serviceName) {
```

Comment:
I think azure service should start with `Azure`

Reply:
this name from user defined, and you cannot assure it start with `Azure`.